components/screen-layout/ErrorScreen.tsx

```diff
@@ -0,0 +1,32 @@
+import { ReactNode } from 'react'
+
+import ContentWithAvatar from '@/components/screen-layout/ContentWithAvatar'
+import { ScreenViewProps } from '@/components/screen-layout/ScreenView'
+import ScreenViewCentered from '@/components/screen-layout/ScreenViewCentered'
+import { useTranslation } from '@/hooks/useTranslation'
+
+type Props = {
+  title?: string
+  text?: string
+  actionButton?: ReactNode
+} & Omit<ScreenViewProps, 'children'>
+
+/**
+ * Error screen with error message and title
+ */
+const ErrorScreen = ({ title, text, actionButton, ...rest }: Props) => {
+  const t = useTranslation('ErrorScreen')
+
+  return (
+    <ScreenViewCentered options={{ headerTransparent: true }} title="" {...rest}>
```

Comment:
Do we still need to provide `title=""`? (Just asking, not mandatory to solve for this PR)